src/Cli/dotnet/ToolPackage/ToolPackageDownloader.cs

```diff
@@ -116,7 +122,7 @@ internal class ToolPackageDownloader : IToolPackageDownloader
 
                     if (package == null)
                     {
-                        DownloadAndExtractPackage(packageLocation, packageId, nugetPackageDownloader, toolDownloadDir.Value, _toolPackageStore, packageVersion, packageSourceLocation).GetAwaiter().GetResult();
+                        DownloadAndExtractPackage(packageLocation, packageId, nugetPackageDownloader, toolDownloadDir.Value, _toolPackageStore, packageVersion, packageSourceLocation, givenSpecificVersion).GetAwaiter().GetResult();
```

Comment:
Including the parameter name here should make this a bit easier to read/understand.

```suggestion
                        DownloadAndExtractPackage(packageLocation, packageId, nugetPackageDownloader, toolDownloadDir.Value, _toolPackageStore, packageVersion, packageSourceLocation, includeUnlisted: givenSpecificVersion).GetAwaiter().GetResult();
```